react-web/src/components/Header/Header.tsx

```diff
@@ -19,6 +23,28 @@ const Header = () => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isLoggedIn]);
 
+  useEffect(() => {
+    setPollForAddress(wallet && address && isLoggedIn)
+  }, [wallet, address, isLoggedIn])
+
+  useDelayedApi(
+    async () => {
+      console.log(wallet, address)
```

Comment:
are we keeping this? I'm just asking to know if we are going to keep console logs for production purposes. If not maybe a "//TODO: remove this"